client/tracks/index.js

```diff
@@ -20,7 +20,9 @@ function recordEvent( eventName, eventProperties ) {
 }
 
 const events = {
-	CONNECT_ACCOUNT_CLICKED: 'wcpay_connect_account_clicked',
+	CONNECT_ACCOUNT_CLICKED: 'wcadmin_wcpay_welcome_finish_setup',
```

Comment:
`window.wcTracks.recordEvent` actually prepends `wcadmin_` to track names, you'll need to remove `wcadmin_` in all our track names here, otherwise it'll be redundant:

![image](https://user-images.githubusercontent.com/3747241/115332866-3be18300-a1cb-11eb-8d83-c3a3dfe2e446.png)

Reply:
Nice catch! I fixed this in 173c388